server/lib/chat/channel-banner-models.ts

```diff
@@ -0,0 +1,134 @@
+import sql from 'sql-template-strings'
+import { assertUnreachable } from '../../../common/assert-unreachable'
+import { ChannelBanner, ChannelBannerId } from '../../../common/chat-channels/channel-banners'
+import db, { DbClient } from '../db'
+import { Dbify } from '../db/types'
+import { getUrl } from '../file-upload'
+import { Patch } from '../leagues/league-models'
```

Comment:
If you're going to start using this then please complete the TODO and move it, it's not okay for this file to depend on Leagues